src/backend/controllers/userController.js

```diff
@@ -1,198 +1,177 @@
-const User = require("../models/User");
-const bcrypt = require("bcryptjs");
-const jwt = require("jsonwebtoken");
+const User = require('../models/User');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
 
 // Register a new user
 exports.registerUser = async (req, res) => {
-    const {
-        firstName,
-        lastName,
-        email,
-        password,
-        isAgent,
-        mobileNumber,
-        profilePic,
-    } = req.body;
-
-    try {
-        // Check if user already exists
-        const userExists = await User.findOne({ email });
-        if (userExists) {
-            return res.status(400).json({ message: "User already exists" });
-        }
-
-        // Hash the password
-        const hashedPassword = await bcrypt.hash(password, 10);
-
-        // Create new user
-        const user = new User({
-            firstName,
-            lastName,
-            email,
-            password: hashedPassword,
-            isAgent: isAgent || false, // Default to false if not provided
-            mobileNumber,
-            profilePic,
-        });
-
-        // Save user to database
-        await user.save();
-
-        res.status(201).json(user);
-    } catch (error) {
-        res.status(500).json({ error: error.message });
+  const { firstName, lastName, email, password, isAgent, mobileNumber } = req.body;
+
+  try {
+    // Check if user already exists
+    const userExists = await User.findOne({ email });
+    if (userExists) {
+      return res.status(400).json({ message: 'User already exists' });
     }
+
+    // Hash the password
+    const hashedPassword = await bcrypt.hash(password, 10);
+
+    // Create new user
+    const user = new User({
+      firstName,
+      lastName,
+      email,
+      password: hashedPassword,
+      isAgent: isAgent || false, // Default to false if not provided
+      mobileNumber
```

Comment:
@Sam559I  I need the profilePic to be added here, so that it can be saved to the DB